src/Components/FilterNavbar/FilterNavbar.jsx

```diff
@@ -77,47 +81,25 @@ export function FilterNavbar() {
                             <ul>
                                 <li>
                                     <input type="radio" name="c1" id="c1" />
-                                    <label for="s1">4 Stars & above</label>
+                                    <label htmlFor="s1">4 Stars & above</label>
                                 </li>
                                 <li>
                                     <input type="radio" name="c1" id="c1" />
-                                    <label for="s2">3 Stars & above</label>
+                                    <label htmlFor="s2">3 Stars & above</label>
                                 </li>
                                 <li>
                                     <input type="radio" name="c1" id="c1" />
-                                    <label for="s3">2 Stars & above</label>
+                                    <label htmlFor="s3">2 Stars & above</label>
                                 </li>
                                 <li>
                                     <input type="radio" name="c1" id="c1" />
-                                    <label for="s4">1 Stars & above</label>
+                                    <label htmlFor="s4">1 Stars & above</label>
```

Comment:
You need to keep the id and htmlFor same so that when you click on the label the radio button gets selected.

Reply:
thank you, very insightful